Library/Homebrew/development_tools.rb

```diff
@@ -10,6 +10,9 @@ def locate(tool)
           path
         elsif File.executable?(path = "/usr/bin/#{tool}")
           Pathname.new path
+        # IF ANDROID
+        elsif File.executable?(path = "/data/data/com.termux/files/usr/bin/#{tool}")
+          Pathname.new path
```

Comment:
Add to https://github.com/Linuxbrew/brew/wiki/Android
```sh
ln -s /data/data/com.termux/files/usr/bin/* ~/.linuxbrew/bin/
```

Reply:
Done.